main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+type Response struct {
```

Comment:
By convention, all the exported stuff should have a description.
```go
// Response - Description here
func Response struct{
```
Do it for all the exported stuff.